db/migrate/20210622151623_generate_users_carts.rb

```diff
@@ -0,0 +1,5 @@
+class GenerateUsersCarts < ActiveRecord::Migration[6.1]
+  def change
+    User.all.each(&:create_cart)
```

Comment:
Czy to jest dobre rozwiązanie, skoro jest już user na produkcji?

Reply:
To jest złe rozwiązanie. Jakbyś miał 1 000 000 userów w bazie to ta migracja moglaby na produkcji nie przejsc. Obczaj find_each :). To też jest dobry materiał na TIL ;)